engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -516,9 +515,10 @@ public void testRawBlockLiteral() throws Exception {
   @Test
   @Ignore
   public void testVariousKindsOfUnicodeWhitespace() throws Exception {
-    // mimics Text_Spec.enso:1049
+    // mimics Text_Spec.enso:1049 and 722
+    // search for: # Disabled in the New Parser
     parseTest("""
-    '\\v\\f\\u{200a}\\u{202f}\\u{205F}\\u{3000}\\u{feff}'.trim
+    t = '\\v\\f\\u{200a}\\u{202f}\\u{205F}\\u{3000}\\u{feff}'.trim
```

Comment:
When debugging this test and comparing the difference between old and new parser I get following:
![obrazek](https://user-images.githubusercontent.com/26887752/200478107-00b006cf-7d7e-43ef-871a-7cf5688e0328.png)
I am not much into Unicode to judge which of the parsers is right, but I am sure we can fix this later.  Hoping for a review by @radeusgd